Nos/Models/Persistence.swift

```diff
@@ -237,8 +237,14 @@ class PersistenceController {
                 let oldEventClause = "(receivedAt <= %@ OR receivedAt == nil)"
                 let notOwnEventClause = "(author.hexadecimalPublicKey != %@)"
                 let readStoryClause = "(isRead = 1 AND receivedAt > %@)"
+                let userReportClause = "(kind == \(EventKind.report.rawValue) AND " +
+                    "authorReferences.@count > 0 AND authorReferences.@count > 0 AND eventReferences.@count == 0)"
```

Comment:
The `eventReferences.@count == 0` part is becuase some event report messages also include a p tag and I assume those cases only mean to report the content not the whole author

Reply:
It looks like the `authorReferences.@count > 0` check is in there twice? We can remove one of them right?
```suggestion
                    "authorReferences.@count > 0 AND eventReferences.@count == 0)"
```